src/main/java/app/controller/soundEngine/CornerAimedTracing.java

```diff
@@ -0,0 +1,36 @@
+package app.controller.soundEngine;
+
+import app.controller.linAlg.Vector;
+import app.model.Map;
+import app.model.agents.Agent;
+
+import java.util.ArrayList;
+
+public class CornerAimedTracing implements SoundEngine{
+    public double compute(Map map, Agent agent){
+        ArrayList<SoundRay> soundRays = new ArrayList<>();
+
+        // Step 1: aim at all the soundsources
+        // right now this means ignore obstactles and return the loudest audible sound from agent
+
+        return map.getSoundSources().stream().map(source -> source.soundLevelFrom(agent.getPosition())).max(Double::compare).get();
```

Comment:
When using streams, the Java convention is to break line at each `.` dot.  i.e.

```java
return map.getSoundSources()
                 .stream()
                 .map(source -> source.soundLevelFrom(agent.getPosition())).max(Double::compare).get();
```

Also, although the lambda with internal method reference is a very compact way of doing this, it is incredibly obfuscated.  Very difficult to read and maintain.  Consider moving the lambda to it's own function that does the actual work.  Try and avoid lambdas with side effects